requirements-base.txt

```diff
@@ -29,7 +30,8 @@ petname==2.6
 phonenumberslite==8.11.3
 Pillow==8.1.0
 progressbar2==3.32.0
-psycopg2-binary==2.7.7
+psycopg2-binary==2.7.7; python_version == '3.6'
+psycopg2-binary==2.8.6; python_version > '3.6'
```

Comment:
This is to help with Python 3.8 & Python 3.9 on Mac and Linux:
https://github.com/getsentry/sentry/pull/24394/checks?check_run_id=2096888344
```
gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -I/usr/local/opt/sqlite/include -I/usr/local/opt/sqlite/include -DPSYCOPG_DEFAULT_PYDATETIME=1 -DPSYCOPG_VERSION=2.7.7 (dt dec pq3 ext lo64) -DPG_VERSION_NUM=130002 -DHAVE_LO64=1 -I/Users/runner/work/sentry/sentry/.venv/include -I/Users/runner/hostedtoolcache/Python/3.8.8/x64/include/python3.8 -I. -I/usr/local/include -I/usr/local/include/postgresql/server -c psycopg/psycopgmodule.c -o build/temp.macosx-10.14-x86_64-3.8/psycopg/psycopgmodule.o
    psycopg/psycopgmodule.c:689:18: error: incomplete definition of type 'struct _is'
        while (interp->next)
               ~~~~~~^
    /Users/runner/hostedtoolcache/Python/3.8.8/x64/include/python3.8/pystate.h:20:8: note: forward declaration of 'struct _is'
    struct _is;
           ^
    psycopg/psycopgmodule.c:690:24: error: incomplete definition of type 'struct _is'
            interp = interp->next;
                     ~~~~~~^
    /Users/runner/hostedtoolcache/Python/3.8.8/x64/include/python3.8/pystate.h:20:8: note: forward declaration of 'struct _is'
    struct _is;
           ^
    2 errors generated.
    
    It appears you are missing some prerequisite to build the package from source.
    
    You may install a binary package by installing 'psycopg2-binary' from PyPI.
    If you want to install psycopg2 from source, please install the packages
    required for the build and try again.
    
    For further information please check the 'doc/src/install.rst' file (also at
    <http://initd.org/psycopg/docs/install.html>).
    
    error: command 'gcc' failed with exit status 1
```